File: src/modules/algorithm/seed/algorithms/array-list/python/remove-solution.py
from exercise import ArrayList

def remove(self, index: int) -> None:
    if 0 <= index < len(self.data):
        for i in range(index, len(self.data) - 1):
            self.data[i] = self.data[i + 1]
        self.data.pop()

ArrayList.remove = remove 

File: src/modules/algorithm/seed/algorithms/array-list/typescript/remove-solution.ts
import { ArrayList } from './exercise';

ArrayList.prototype.remove = function (index) {
  if (index >= 0 && index < this.data.length) {
    for (let i = index; i < this.data.length - 1; i++) {
      this.data[i] = this.data[i + 1];
    }
    this.data.pop();
  }
};
